catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -21,6 +21,14 @@
             name="Directory Path" id="monitoredDirectoryPath" required="true"
             type="String" default=""/>
 
+        <AD description="Specifies the number of threads to use when monitoring"
```

Comment:
This doesn't really tell me anything.  Does this mean that the number I put here will be the maximum number of concurrent files to be processed within a directory? 

An admin should not have to know about "threads" but rather what the result of changing this values means from a processing perspective. Also, if the maximum is 8 and it is hardcoded - I would at that to this description.